docs_src/pipelinebenchmarking.md

```diff
@@ -129,6 +130,28 @@ sudo ./benchmark.sh --stream_density <target FPS> --logdir <output dir>/data --i
 **_NOTE:_** Because stream density requires a continuous video stream it is recommended to use an RTSP stream, USB camera, or RealSense camera. If these options are not available you can use the [camera simulator](#appendix-benchmark-helper-scripts) to continuously loop through a video file as an RTSP stream.
 
 ---
+
+#### Workload
+We are currently supporting 2 types of workload:
+    1. dlstreamer
+    2. opencv-ovms
+
+These are the input value for `--workload` parameter for benchmark.sh script. The default value for `--workload` parameter is `dlstreamer` in case it is not provided when running benchmark.sh script.
+
+#### Benchmark Specified Profile for Opencv-ovms
+For running Opencv-OVMS workload, we are supporting different programming languages and different models. You may specify [language choice](./OVMS/supportingDifferentLanguage.md) and [model input](./OVMS/supportingDifferentModel.md). Then you may **prefix** benchmark script run command with specific profile.
```

Comment:
```suggestion
For running OpenCV-OVMS workload, we are supporting different programming languages and different models. You may specify [language choice](./OVMS/supportingDifferentLanguage.md) and [model input](./OVMS/supportingDifferentModel.md). Then you may **prefix** benchmark script run command with specific profile.
```
Follow same convention everywhere

Reply:
opencv-ovms all lower case to match the worload value

Reply:
Keep the naming convention consistent across the documentation, don't change it.